lib/kitchen/directions/bake_chapter_summary.rb

```diff
@@ -26,7 +26,11 @@ def bake(chapter:, metadata_source:, uuid_prefix: '.', klass: 'summary')
 
           # TODO: include specific page types somehow without writing it out
           chapter.non_introduction_pages.each do |page|
-            summary = page.summary
+            begin
+              summary = page.summary
+            rescue
+              next
+            end
```

Comment:
Is this the best way to do this? #summary raises a Recipe error when it can't find anything, but the other search options seemed like they'd eat up a lot of time and might not use the right selectors.

Reply:
`next unless page&.summary` would also accomplish this i believe

Reply:
That only catches it if page is nil-- #summary raises it's own error by using #first!